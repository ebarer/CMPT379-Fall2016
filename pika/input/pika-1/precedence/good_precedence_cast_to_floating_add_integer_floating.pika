exec {

  print [ 32 | float ] + 2.0, _n_.
  print [ 42 | float ] + 8.2, _n_.
  print [ 66 | float ] + 6.7, _n_.
  print [ 87 | float ] + 1.0, _n_.
  print [ 99 | float ] + 4.4, _n_.
  print [ 54 | float ] + 9.3, _n_.
  print [ 19 | float ] + 3.0, _n_.
  print [ 19 | float ] + 2.0, _n_.
  print [ 10 | float ] + 7.1, _n_.
  print [ 43 | float ] + 2.1, _n_.
  print [ 96 | float ] + 9.6, _n_.
  print [ 92 | float ] + 8.2, _n_.

  print 9.0 + [ 64 | float ], _n_.
  print 1.4 + [ 45 | float ], _n_.
  print 9.4 + [ 93 | float ], _n_.
  print 4.3 + [ 23 | float ], _n_.
  print 2.2 + [ 57 | float ], _n_.
  print 8.7 + [ 77 | float ], _n_.
  print 3.4 + [ 31 | float ], _n_.
  print 8.0 + [ 98 | float ], _n_.
  print 5.7 + [ 51 | float ], _n_.
  print 9.6 + [ 16 | float ], _n_.
  print 4.6 + [ 5 | float ], _n_.
  print 2.1 + [ 49 | float ], _n_.

  print [ 58 | float ] + [ 88 | float ], _n_.
  print [ 14 | float ] + [ 41 | float ], _n_.
  print [ 20 | float ] + [ 33 | float ], _n_.
  print [ 77 | float ] + [ 73 | float ], _n_.
  print [ 11 | float ] + [ 13 | float ], _n_.
  print [ 37 | float ] + [ 39 | float ], _n_.
  print [ 55 | float ] + [ 26 | float ], _n_.
  print [ 20 | float ] + [ 30 | float ], _n_.
  print [ 16 | float ] + [ 99 | float ], _n_.
  print [ 46 | float ] + [ 32 | float ], _n_.
  print [ 13 | float ] + [ 41 | float ], _n_.
  print [ 89 | float ] + [ 52 | float ], _n_.

}

