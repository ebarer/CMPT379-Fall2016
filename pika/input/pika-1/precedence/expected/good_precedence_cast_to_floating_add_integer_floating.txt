34
50.2
72.7
88
103.4
63.3
22
21
17.1
45.1
105.6
100.2
73
46.4
102.4
27.3
59.2
85.7
34.4
106
56.7
25.6
9.6
51.1
146
55
53
150
24
76
81
50
115
78
54
141
